doc/cli/nmo-import-mongo.md

```diff
@@ -0,0 +1,24 @@
+nmo-import-mongo(1) -- Bulk import a MongoDB collection
+===========================================
+
+## SYNOPSIS
+
+    nmo import-mongo [clustername] [database] [MongoDB-url] [collection]
+    nmo import-mongo [url] [database] [MongoDB-url] [collection]
```

Comment:
i think those are all required arguments?

(use `< >` instead of `[ ]`, the latter are to signal optional arguments)